File: src/main/java/com/vjiki/music/dto/CreateChatRequest.java
package com.vjiki.music.dto;

import java.util.ArrayList;
import java.util.List;
import java.util.UUID;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CreateChatRequest {
    private String type = "DIRECT";
    private String title;
    private String description;
    private String avatarUrl;
    private UUID ownerId;
    private List<UUID> participantIds = new ArrayList<>();
    private Boolean isEncrypted = false;
}

File: bin/main/com/vjiki/music/dto/CreateChatRequest.kt
package com.vjiki.music.dto

import java.util.UUID

data class CreateChatRequest(
    val type: String = "DIRECT",
    val title: String? = null,
    val description: String? = null,
    val avatarUrl: String? = null,
    val ownerId: UUID? = null,
    val participantIds: List<UUID>,
    val isEncrypted: Boolean = false
)
